src/main/java/blackjack/domain/Player.java

```diff
@@ -0,0 +1,37 @@
+package blackjack.domain;
+
+public class Player {
+
+    private final String name;
+    protected final Hand hand;
```

Comment:
```suggestion
   final Hand hand;
```
protected 를 붙이고 안 붙였을 때 어떤 차이가 있을까요?

Reply:
붙인다면 자식 클래스에서 해당 필드에 직접 접근이 가능합니다. 
Dealer(자식 클래스)에서 hand 계산 메서드에 접근하기 위해 사용했었는데 private 을 사용하더라도 사용이 가능하다는 것을 뒤늦게 알아챘네요. Player 의 캡슐화와 구현 세부 사항을 노출시키지 않기 위해 바꾸도록 하겠습니다!

Reply:
protected 를 붙이지 않고 package-private 로 해도 자식 클래스에서 접근 가능해요!
차이는 그 자식 클래스가 같은 패키지에 있는가, 외부 패키지에 있는가 거든요. 해당 질문은 package-private 이라는 접근제한자도 있다는걸 알려드리고 싶어서 남긴 리뷰였어요. 그래도 좋은 방향으로 이해하시고 잘 진행해주신 것 같네요 :)

Reply:
기본 접근 제어자를 말씀하셨던 거군요. 사용해보는 것을 추천하셨던걸까요?
패키지를 잘 나누더라도 같은 패키지 내부에서 접근이 가능하여 오용하기 쉽지않을까 염려됩니다.